### ISSUE TITLE
Docs updates for: xERC20

### ISSUE THREAD

**Issue by just-a-node** [connext/monorepo#5334]
## Background

- [ ] Add notes on migration procedure and caveats for Category C tokens
- [ ] Add xERC20 factory addresses

**Comment by preethamr** [comment]
@prathmeshkhandelwal1 is this still needed (not done )?

**Comment by prathmeshkhandelwal1** [comment]
Nope, we need to prio this and add factory addresses and common adapter address.
